src/components/Filter/filter.module.css

```diff
@@ -0,0 +1,17 @@
+.filter_container {
+  display: flex;
+  top: 5rem;
+  position: absolute;
+  gap: 2rem;
+  padding-left: 3rem;
+  flex-wrap: wrap;
+}
+.filter_container button {
```

Comment:
Suggestion : When you write more than one selector then be sure about specificity and inheritance